(chore): storage section feedback (#5917)
aws-amplify/docs: src/fragments/lib-v1/storage/js/download.mdx

> Following are the ways of getting data that was previously uploaded:

Reviewer: Reads a little awkwardly to me, just a suggestion

```suggestion
The following options are available for retrieving previously uploaded data:
```
